src/lib/logs/log_self_monitoring.py

```diff
@@ -61,11 +61,15 @@ def put_sfm_into_queue(context: LogsContext):
 
 
 async def create_sfm_worker_loop(sfm_queue: Queue, logging_context: LoggingContext, instance_metadata: InstanceMetadata):
+    sfm_loop = asyncio.get_event_loop()
```

Comment:
Is this really sfm_loop, like only for sfm? Or is it some main loop that you reuse here. If so, then the naming is off

Reply:
Right. Main loop